caput/astro/observer.py

```diff
@@ -231,35 +174,46 @@ def unix_to_lsa(self, time):
     lsa = unix_to_lsa
 
     @arraytools.listize()
+    @overload
+    def lsa_to_unix(self, lsa: np.floating, time0: np.floating) -> np.floating: ...
+    @overload
+    def lsa_to_unix(
+        self, lsa: npt.ArrayLike[np.floating], time0: npt.ArrayLike[np.floating]
+    ) -> np.ndarray[np.floating]: ...
     def lsa_to_unix(self, lsa, time0):
         """Convert a Local Stellar Angle (LSA) on a given day to a UNIX time.
 
         Parameters
         ----------
-        lsa : scalar or np.ndarray
+        lsa
             Local Earth Rotation Angle degrees to convert.
```

Comment:
```suggestion
            Local Stellar Angle to convert, in degrees.
```
We've explained the relationship between LSA and ERA above.